.github/workflows/deploy.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           TF_VAR_google_tag_manager_environment_preview: ${{ secrets.STAGING_GOOGLE_TAG_MANAGER_ENVIRONMENT_PREVIEW }}
         run: |
           script/deploy-terraform
-        if: ${{ github.ref == "refs/heads/develop" }}
+        if: github.ref == 'refs/heads/develop'
```

Comment:
I assume it's the single quotes that are significant here (the `${{ .. }}` is optional for conditionals..?

Reply:
Aye, I just removed the `${{ ... }}` while I was there, as I couldn't find many examples of them in the wild.